docs/administration-guide/gaffer-config/graph-metadata.md

```diff
@@ -0,0 +1,97 @@
+# Graph Metadata Configuration
+
+The graph configuration file is a JSON file that configures few bits of the
+Gaffer graph. Primarily it is used to set the name and description along with
+any additional hooks to run before an operation chain e.g. to impose limits on
+max results etc. For the example a simple graph configuration file might look
+like:
+
+```json title="graphConfig.json"
+{
+    "graphId": "ExampleGraph",
+    "description": "An example graph"
+}
+```
+
+## Configuring a Standard Deployment
+
+To change any of the values for a standard Gaffer deployment all thats needed
+is to configure the JSON file for the `graphConfig`. The key value pairs in
+the file can then be configured as you wished and upon restarting the graph
+the values will be updated (assuming the file is loaded correctly).
+
+The standard file location in the gaffer images for the file is `/gaffer/graph/graphConfig.json`
+
+## Configuring a Helm Deployment
+
+Configuring the graph metadata via Helm follows a similar principal to the JSON
+files however, you must use the YAML format instead for the key value pairs. The
+following gives an example of how the description value can be updated via Helm.
+
+Create a file called `graph-meta.yaml`. We will use this file to add our
+description and graph ID. Changing the description is as easy as changing the
+`graph.config.description` value.
+
+```yaml
+graph:
+  config:
+    description: "My graph description"
+```
+
+Upgrade your deployment using Helm to load the new file:
+
+```bash
+helm upgrade my-graph gaffer-docker/gaffer -f graph-metadata.yaml --reuse-values
+```
+
+The `--reuse-values` argument means we do not override any passwords that we set
+in the initial construction.
+
+!!! tip
+    You can see you new description if you to the Swagger UI and call the
+    `/graph/config/description` endpoint.
+
+## Updating the Graph ID
```

Comment:
I appreciate this may have been taken from a K8s related page, but this section is mixing general Gaffer info together with helm specific config.

Reply:
That's fair, I think you have a few other comments on this. I merged the existing info about configuring a k8s deployment into these pages under graph configuration, the logic being that a reader would want to know how to configure say the graph metadata, so they go to the page about configuring the graph metadata which ideally should cater for all possible deployments. The alternative is that might end up duplicating the information but just on k8s specific pages as you are configuring the same thing just in a slightly different way.